### DISCUSSION TITLE
How to make package for AdonisJs v5?

### DISCUSSION THREAD

**Discussion by feniljariwala82** [adonisjs/core#3403] [Help] [answered]
I want to write a wrapper package for Razorpay for AdonisJs. I looked into the documentation but I could not found any way to build package for AdonisJs. If I'm making any research mistake, please guide me. Or provide me some information about making package for AdonisJs v5. Thank you.

**Comment by thetutlage** [comment, marked as answer]
There isn't any documentation for it yet. However, you can use any package of AdonisJS as an inspiration to create yours. A good one will be the redis package, as it itself is a wrapper on top of ioredis

**Reply by feniljariwala82** [reply]
Thanks for replying. I am looking into it. Thanks.
